tests/devices/unit_tests/test_attenuator.py

```diff
@@ -0,0 +1,35 @@
+from unittest.mock import MagicMock
+
+import numpy as np
+import pytest
+from ophyd.sim import make_fake_device
+from ophyd.status import Status
+
+from dodal.devices.attenuator import Attenuator
+
+CALCULATED_VALUE = np.random.randint(0, 10, size=16)
```

Comment:
Should: Sorry, we shouldn't use random numbers here. Randomness in unit tests is generally bad as it causes repeatability issues